integrations/docker/build.sh

```diff
@@ -91,9 +96,16 @@ docker image prune --force
     }
 }
 
+#Fix storage location of Docker images and containers for osx or linux
```

Comment:
drop this comment

Reply:
The comment will be deleted. Thanks for the review :)